contracts/Compliance.sol

```diff
@@ -0,0 +1,40 @@
+pragma solidity 0.5.16;
+
+import "@openzeppelin/contracts/ownership/Ownable.sol";
+import "./spec_interfaces/ICompliance.sol";
+
+contract Compliance is ICompliance, Ownable {
+
+    IContractRegistry contractRegistry;
```

Comment:
Why does it need a contract registry if nobody is using it?

and, if we later decide that this is a requirement for all contracts then maybe we should separate this into an interface or a super implemenration such as `RegisteredContract`: 
```
event ContractRegistryChanged(oldRegistry, newRegistry);

setContractRegistry(newRegisrtry)
getRegistryName() returns string
isRegistered() returns bool
...
```
and all our contracts can inherit from this contract..?

but if we don't have this standard, why not exclude this to avoid questions like - what kind of event is fired when we replace the regisrty? etc...

@OdedWx, @vistra what do you think ?

Reply:
It will be used when this contract sends notifications on compliance changes.
Agree we can use a super implementation, but not sure about the specifics. Anyway this is out of scope for this PR I think (this pattern is used across all contracts).

Reply:
why not send the notifications now?

Reply:
I didn't want to touch the elections contract to avoid collisions with you as we discussed. I also prefer not adding nops as they create an illusion of integration. We will have it in the next 3 days anyway.